frontend/packages/db/prisma/migrations/20250327033040_add_branch_name_to_overall_review/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `branchName` to the `OverallReview` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "OverallReview" ADD COLUMN     "branchName" TEXT NOT NULL;
```

Comment:
The migration will probably fail and should be handled after the merge! I think I can reset the database.